resources/ts/Pages/Integrations/New.tsx

```diff
@@ -36,6 +37,10 @@ const New = ({ subscriptions }: Props) => {
       subscription.price === 0
   )?.id;
 
+  const basicSubscriptionIds = subscriptions
+    .filter((subscription) => subscription.category.includes("Basic"))
```

Comment:
Normally you can use the type/constant SubscriptionCategory to check the category. No custom check on the `Basic` string